docs/docs/pyproject.md

```diff
@@ -293,3 +293,9 @@ build-backend = "poetry.core.masonry.api"
 
     If your `pyproject.toml` file still references `poetry` directly as a build backend,
     you should update it to reference `poetry_core` instead.
+
+## private
```

Comment:
This should probably go before the PEP-517 section.

Reply:
Moved